### PR TITLE
Show track scrubber in playlist context

### PR DESCRIPTION
**Pull request by Dananji**
Related issue: https://github.com/avalonmediasystem/avalon/issues/5624

### CODE REVIEW

File: src/components/MediaPlayer/VideoJS/components/js/VideoJSTrackScrubber.js [hunk 1]
```diff
@@ -201,19 +208,25 @@ function TrackScrubberButton({ player, trackScrubberRef, timeToolRef }) {
 	 */
 	const timeUpdateHandler = () => {
 		if (player.isDisposed()) return;
```
Comment (masaball) [inline src/components/MediaPlayer/VideoJS/components/js/VideoJSTrackScrubber.js:210]: I did not think to test the track scrubber when I was looking at the issue with the flashing struct nav. Just to be safe that we don't accidentally reintroduce that behavior, we should probably have this time update return on ended as well.
```suggestion
		if (player.isDisposed() || player.ended()) return;
```

File: src/components/MediaPlayer/VideoJS/components/js/VideoJSTrackScrubber.js [hunk 2]
```diff
@@ -24,15 +25,19 @@ class VideoJSTrackScrubber extends vjsComponent {
...
-		player.ready(() => {
-			this.mount();
-		});
+		/* When player is ready and the trackScrubber element is initialized,
+		call method to mount React component.
+		*/
+		if (this.options.trackScrubberRef.current) {
+			player.ready(() => {
+				this.mount();
+			});
```
Comment (masaball) [inline src/components/MediaPlayer/VideoJS/components/js/VideoJSTrackScrubber.js:34]: It looks like everything in this file is indented at least twice as far as it should be for some reason?

File: src/components/MediaPlayer/VideoJS/components/styles/VideoJSTrackScrubber.scss [hunk 3]
```diff
@@ -15,11 +15,19 @@
...
 	justify-content: space-between;
 
+	@media (max-width: 680px) {
+		width: 99.65%;
+	}
+
+	@media (min-width: 1081px) {
+		width: 99.85%;
+	}
+
```
Comment (masaball) [inline src/components/MediaPlayer/VideoJS/components/styles/VideoJSTrackScrubber.scss:30]: This file also has double width indents.

File: src/components/MediaPlayer/MediaPlayer.test.js [hunk 4]
```diff
@@ -46,7 +46,7 @@ describe('MediaPlayer component', () => {
     });
   });
 
-  describe('with video manifest', () => {
+  describe('with a regualr video Manifest', () => {
```
Comment (masaball) [inline src/components/MediaPlayer/MediaPlayer.test.js:49]:
```suggestion
  describe('with a regular video Manifest', () => {
```

**Review by masaball — COMMENTED** [review]
Looks good for the most part. Have one suggestion that may prevent a regression and then some styling questions.

File: src/components/MediaPlayer/VideoJS/components/js/VideoJSTrackScrubber.js (hunk 2 above)
Comment (Dananji) [reply src/components/MediaPlayer/VideoJS/components/js/VideoJSTrackScrubber.js]: It was formatting issue when I created the files. Thanks for catching that. I wasn't seeing that in my editor with its settings.

**Review by masaball — APPROVED** [review]

**Pull request merged**